### ISSUE TITLE
something about SMASH

### ISSUE THREAD

**Issue by shuida** [tata1661/FewShotPapers#1]
SMASH is about NAS rather than FSL. "one-shot" in NAS means  training only one model. It's nothing about the number of training data.

**Comment by tata1661** [comment]
This depends on how you understand sample. 

In NAS, I regard one run of the model as one training sample. 
In classic NAS methods such as grid search, random search and Bayes optimization, one must run the model several times, obtain its performance on validation set, then decide which is the best setting.
In contrast, SMASH only needs one run.

That is why I classify SMASH into FSL.
